docs/config-app.md

```diff
@@ -206,6 +206,9 @@ For `console` backend type available next options:
 - `metrics.console.enabled` - if equals to `true` then `console` will be used to submit metrics.
 - `metrics.console.interval` - interval in seconds between successive sending metrics.
 
+For `prometheus` backend type available next options:
+- `metrics.prometheus.port` - if a port is specified greater `0` a prometheus reporter will start on that port 
```

Comment:
As we can see from code
`@Configuration
    @ConditionalOnProperty(prefix = "metrics.prometheus", name = "port")
    static class PrometheusServerConfiguration {`
It will start if any `metrics.prometheus.port` is specified